doc/apps/flask.md

```diff
@@ -0,0 +1,29 @@
+# Flask
+
+To deploy a Flask application in Ploomber Cloud you need:
+
+- A `Dockerfile`
+- Your code
+## `Dockerfile`
+
+You need to provide a `Dockerfile`, you can use this [template](https://github.com/ploomber/doc/blob/main/examples/docker/flask/Dockerfile) to get started. The template contains the minimal steps needed for a deployment but you need to modify so it installs any required dependencies and copies your code into the Docker image.
```

Comment:
This page looks similar to the Docker one?

Reply:
yeah, that's on purpose. for the docker one we have to choose a framework and currently, I've only got flask working